prow/config.yaml

```diff
@@ -592,28 +592,28 @@ presubmits:
   - name: pull-security-kubernetes-kubemark-e2e-gce
     always_run: true
     context: pull-security-kubernetes-kubemark-e2e-gce
-    rerun_command: "@k8s-bot pull-security-kubernetes-kubemark-e2e-gce test this"
-    trigger: "@k8s-bot (pull-security-kubernetes-kubemark-e2e-gce |kubemark e2e )?test this"
+    rerun_command: "/test pull-security-kubernetes-kubemark-e2e-gce"
+    trigger: "(^@k8s-bot (pull-security-kubernetes-kubemark-e2e-gce )?test this\r?$|^/test( pull-security-kubernetes-kubemark-e2e-gce)?\r?$)"
 
   - name: pull-security-kubernetes-kubemark-e2e-gce-gci
     context: pull-security-kubernetes-kubemark-e2e-gce-gci
-    rerun_command: "@k8s-bot pull-security-kubernetes-kubemark-e2e-gce-gci test this"
-    trigger: "@k8s-bot pull-security-kubernetes-kubemark-e2e-gce-gci test this"
+    rerun_command: "/test pull-security-kubernetes-kubemark-e2e-gce-gci"
+    trigger: "(^@k8s-bot pull-security-kubernetes-kubemark-e2e-gce-gci test this\r?$|^/test pull-security-kubernetes-kubemark-e2e-gce-gci\r?$)"
 
   - name: pull-security-kubernetes-node-e2e
     always_run: true
     context: pull-security-kubernetes-node-e2e
-    rerun_command: "@k8s-bot pull-security-kubernetes-node-e2e test this"
-    trigger: "@k8s-bot (pull-security-kubernetes-node-e2e |node e2e )?test this"
+    rerun_command: "/test pull-security-kubernetes-node-e2e"
+    trigger: "(^@k8s-bot (pull-security-kubernetes-node-e2e )?test this\r?$|^/test( pull-security-kubernetes-node-e2e)?\r?$)"
 
   kubernetes/test-infra:
   - name: pull-test-infra-bazel
     context: pull-test-infra-bazel
     branches:
     - master
     always_run: true
-    rerun_command: "@k8s-bot pull-test-infra-bazel test this"
-    trigger: "@k8s-bot (pull-test-infra-bazel |bazel )?test this"
+    rerun_command: "@k8s-bot bazel test this"
+    trigger: "^@k8s-bot bazel test this\r?$"
```

Comment:
Something is very broken if this has `always_run: true`, does not trigger on `@k8s-bot test this` and yet unit tests pass!